scripts/mmteb/task_selection/get_table_from_hf.py

```diff
@@ -0,0 +1,145 @@
+import json
+import os
+
+# supress warnings
+import warnings
+from functools import reduce
+
+import pandas as pd
+from huggingface_hub import HfApi, hf_hub_download, snapshot_download
+from huggingface_hub.repocard import metadata_load
+from yaml import safe_load
+
+warnings.filterwarnings("ignore")
+
+HF_TOKEN = "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"
```

Comment:
Just double checking that you've invalidated this token.

Reply:
Oh yea, It is actually not mine, but Vaibhav's. I wrote him that he had accidentally pushed it and it should have been invalidated. Have also just pushed a change where I remove it as well

Reply:
cc @vaibhavad (in case it wasn't deleted)